File: python/prism_sdk/local_projection_integrity_contract_model.py
"""Graph P32 local contract_model projection-integrity feature F05."""
from .projection_integrity_support import ProjectionRequest4,ProjectionCard7,ProjectionIntegrityError,manifest,qualify
FEATURE_ID="AFA-graph-P32-F05";CONTRACT_VERSION="graph-local_projection_integrity_contract_model/1.0"
def local_projection_integrity_contract_model_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="contract_model")
def qualify_local_projection_integrity_contract_model(request:ProjectionRequest4)->ProjectionCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="contract_model")

File: crates/graph/src/local_projection_integrity_contract_model.rs
//! Graph P32 local contract_model projection-integrity feature.
use super::projection_integrity_support::{manifest,qualify,ProjectionCard7,ProjectionIntegrityError,ProjectionRequest4};
pub const FEATURE_ID:&str="AFA-graph-P32-F05";pub const CONTRACT_VERSION:&str="graph-local_projection_integrity_contract_model/1.0";
pub fn local_projection_integrity_contract_model_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"local","contract_model")}
pub fn qualify_local_projection_integrity_contract_model(request:&ProjectionRequest4)->Result<ProjectionCard7,ProjectionIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"local","contract_model")}
